crates/astria-sequencer-relayer/src/relayer/write/conversion.rs

```diff
@@ -71,7 +71,7 @@ impl Submission {
     // significance.
     #[allow(clippy::cast_precision_loss)]
     pub(super) fn compression_ratio(&self) -> f64 {
-        self.compressed_size() as f64 / self.uncompressed_size() as f64
+        self.uncompressed_size() as f64 / self.compressed_size() as f64
```

Comment:
maybe we should add a comment saying this is now uncompressed:compressed?

Reply:
I moved this change to [a separate PR](https://github.com/astriaorg/astria/pull/1075) and added a comment there.